src/Config/ProjectConfig.php

```diff
@@ -30,8 +33,13 @@ function __construct(int $id, string $name, array $variations)
     public static function fromArray(array $data): ProjectConfig
     {
 
-        $id   = $data['id'] ?? 0;
-        $name = $data['name'] ?? '';
+        $id    = $data['id'] ?? 0;
+        $name  = $data['name'] ?? '';
+        $state = ProjectState::PAUSED;
+
+        if ('active' === ($data['state'] ?? null)) {
```

Comment:
This is being done in multiple places, Perhaps we should move it into ProjectState? ProjectState::fromString(string $value)?

Reply:
fixed